dbt_platform_helper/constants.py

```diff
@@ -1,6 +1,10 @@
+# Todo: Move to Config provider
 PLATFORM_CONFIG_FILE = "platform-config.yml"
+# Todo: Can we get rid of this yet?
 PLATFORM_HELPER_VERSION_FILE = ".platform-helper-version"
```

Comment:
@antroy-madetech ?

Reply:
Yes, I think so. Any teams still using it will be forced to upgrade, so it'll be a breaking change.

Reply:
I'm not clear from your answer whether you think we should keep support for this file?